### DISCUSSION TITLE
Is there a way to find previous note?

### DISCUSSION THREAD

**Discussion by mkasu** [SilentVoid13/Templater#249] [Help]
Hi. I regularly take meeting notes like "2021-05-27 ABC Meeting". I would like to create a navigation link to the previous ABC Meeting note (which would be somewhere in the past but usually not yesterday), e.g. to [[2021-05-20 ABC Meeting]]. 

Is there some built-in way how I can get the most recent note with the same title? If not, how would I approach this with a custom function? Thanks!

**Comment by AB1908** [comment]
Hmm, I think the most efficient way would be to get the parent folder with `app.vault.getAbstractFileByPath` and then select the last child after sorting.
